### ISSUE TITLE
Feature request: open/create notes in app by default

### ISSUE THREAD

**Issue by disser** [drgrib/alfred-bear#48]
The default behavior of most actions is to open or create notes in a new window by default. I like mine to open in the app by default to minimize screen clutter, so I've reworked the workflows that way.  It would be nice to have a configuration of some sort to allow the default to be user selectable without changing the workflows.

**Comment by drgrib** [comment]
I looked into this a bit and, given the way Alfred uses hotkeys, this is currently impossible in a clean way as far as I can tell. 

Currently, notes are opened in the Bear app by holding down the command key as you open the note. Ideally, I could provide a preference file where you could change the behavior of modifier keys. But Alfred doesn't allow this. Modifier keys can't be edited programmatically (e.g. through preference file values), only through manually changing them in the Alfred GUI.

I could make it to where both no modifier and command key open in the Bear app but I don't like this solution. For now, the best solution for your situation appears to be what you've already done. If Alfred changes the design so that modifier keys can be remapped programmatically, I plan on adding a preference file for setting them.

**Comment by konstk1** [comment]
I would like this to be the default option (open inside the main Bear window). This is how Chris Brown's workflow worked. Opening both Bear and the note in a separate window seems clunky and cluttered.

**Comment by Anber55** [comment]
Excuse me, did you implement this function (opening in the bear palm master window), I also hope so, please can share the work flow, thank you

**Comment by drgrib** [comment]
No, this is still a limitation on Alfred's end, as far as I know.

**Comment by drgrib** [comment]
> Opening both Bear and the note in a separate window seems clunky and cluttered.

I feel exactly the opposite. To me, having to constantly click around in Bear is inefficient and slow. Being able to open 2 or more notes to cross reference and transfer between them without an entire note navigation interface cluttering the workspace is more efficient.
